wiki/Glossary/en.md

```diff
@@ -755,9 +755,9 @@ Stacking hit circles (a clear violation of beat spacing) is, nevertheless, permi
 
 ### Standard
```

Comment:
the whole section needs to be removed -- it is essentially describing the thing we are removing right now